TMS.Net07.Homework.GeometricFigures/Program.cs

```diff
@@ -0,0 +1,203 @@
+                            secindResultToConvertForFirstPoint = int.TryParse(Console.ReadLine(), out SecondCoordinateForFirstPoint);
+
+                            Console.WriteLine("Second Point: ");
+                            firstResultToConvertForSecondPoint = int.TryParse(Console.ReadLine(), out FirstCoordinateForSecondPoint);
+                            secindResultToConvertForSecondPoint = int.TryParse(Console.ReadLine(), out SecondCoordinateForSecondPoint);
+                            
+                            var hexagon = new Hexagon(new Point(FirstCoordinateForFirstPoint, SecondCoordinateForFirstPoint),
+                                new Point(FirstCoordinateForSecondPoint, SecondCoordinateForSecondPoint));
+
+                            ShapeManager.PrintShapePerimeter(hexagon);
+                            ShapeManager.PrintShapeSquare(hexagon);
+                            Draw(hexagon);
+                            Console.ReadLine();
+                            Console.Clear();
+                            break;
+                        case 5:
+                            Console.WriteLine("Circle!");
+                            Console.WriteLine($"Enter coordinates for two points:{Environment.NewLine}" +
+                                                           $"First point: ");
+                            firstResultToConvertForFirstPoint = int.TryParse(Console.ReadLine(), out FirstCoordinateForFirstPoint);
+                            secindResultToConvertForFirstPoint = int.TryParse(Console.ReadLine(), out SecondCoordinateForFirstPoint);
+
+                            Console.WriteLine("Second Point: ");
+                            firstResultToConvertForSecondPoint = int.TryParse(Console.ReadLine(), out FirstCoordinateForSecondPoint);
+                            secindResultToConvertForSecondPoint = int.TryParse(Console.ReadLine(), out SecondCoordinateForSecondPoint);
+
+                            var circle = new Circle(new Point(FirstCoordinateForFirstPoint, SecondCoordinateForFirstPoint),
+                                new Point(FirstCoordinateForSecondPoint, SecondCoordinateForSecondPoint));
+
+                            ShapeManager.PrintShapePerimeter(circle);
+                            ShapeManager.PrintShapeSquare(circle);
+                            Draw(circle);
+                            Console.ReadLine();
+                            Console.Clear();
+                            break;
+                    }
+                }
+                else
+                {
+                    Console.WriteLine("Enter correct number");
+                    Console.ReadLine();
+                    Console.Clear();
+                }
+            }
+        }
+        static void Draw(Shapes shape)
+        {
+            Drawer drawer = new ConsoleEngDescriptionDrawer();
```

Comment:
```suggestion
            Drawer drawer;
```
This initialization doesn't make sense if user's choise is "ru". It's better to create object where it is used (in the corresponding `if` branch in our case)